### PR TITLE
Fix submitForm options behaviour

### PR DESCRIPTION
**Pull request by Deuteu**
Consider binding options and default options for submitForm behaviour

### CODE REVIEW

**Comment by Deuteu** [comment]
@pawelad You should be interested in that PR given gitter chat historic.

File: jquery.confirm.js [hunk 1]
```diff
@@ -69,7 +69,10 @@
         // Default options
         var settings = $.extend({}, $.confirm.options, {
             confirm: function () {
-                if (dataOptions.submitForm){
+                if (dataOptions.submitForm
+                    || (typeof dataOptions.submitForm == "undefined" && options.submitForm)
+                    || (typeof dataOptions.submitForm == "undefined" && typeof options.submitForm == "undefined" && $.confirm.options.submitForm)
+                ) {
```
Comment (mnapoli) [inline jquery.confirm.js]: Could this be simplified to `dataOptions.submitForm || options.submitForm || $.confirm.options.submitForm` ?

Comment (Deuteu) [inline jquery.confirm.js]: You want to make sure that dataOptions override options and both of them override $.confirm.options

(short answer, i'm writing an example and will edit to make my self clear)

Comment (mnapoli) [inline jquery.confirm.js]: I think I understand what you mean. I think `typeof dataOptions.submitForm == "undefined"` can be simplified to `!!dataOptions.submitForm`?

Comment (mnapoli) [inline jquery.confirm.js]: Fair enough I'll be merging then

**Comment by mnapoli** [comment]
I've tagged a new release thank you

**Comment by Deuteu** [comment]
In the same spirit, I've made an other branch for post option, do you want it too ?
(Cf patch-3 on my fork repo)

**Pull request merged**